state/state_test.go

```diff
@@ -337,12 +340,16 @@ func TestProposerFrequency(t *testing.T) {
 func genValSetWithPowers(powers []int64) *types.ValidatorSet {
 	size := len(powers)
 	vals := make([]*types.Validator, size)
+	totalVotePower := int64(0)
 	for i := 0; i < size; i++ {
+		totalVotePower += powers[i]
 		val := types.NewValidator(ed25519.GenPrivKey().PubKey(), powers[i])
 		val.ProposerPriority = cmn.RandInt64()
 		vals[i] = val
 	}
-	return types.NewValidatorSet(vals)
+	valSet := types.NewValidatorSet(vals)
+	valSet.RescalePriorities(totalVotePower)
```

Comment:
`NewValidatorSet` already calls `IncrementProposerPriority` which calls `RescalePriorities`. Do we need to call it twice here?

Reply:
The tests which I believe checks against Chris's proofs, requires RescalePriorities w/ totalVotingPower, not 2*maxVotingPower.

Reply:
> not 2*maxVotingPower

What is this referring to?